School.js

```diff
@@ -12,37 +14,46 @@ class School {
   /**
    * Add class to classes
    * 
-   * @param {string} name - Name of the class
-   * @param {string} teacher - Name of instructor 
-   * @return {Class} Class object
+  //  * @param {string} name - Name of the class
+  //  * @param {string} teacher - Name of instructor 
+  //  * @return {Class} Class object
    */
   addClass(name, teacher) {
     let newClass = new Class(name, teacher);
-    this.classes[name] = newClass;
+    return this.classes[name] = newClass;
   }
 
   /**
    * Enroll student in class
    * 
-   * @param {string} className - Name of the class
-   * @param {Student} student - Student object
-   * @return {Student} Enrolled student
+  //  * @param {string} className - Name of the class
+  //  * @param {Student} student - Student object
+  //  * @return {Student} Enrolled student
    */
-  enrollStudent(className, student) {
-    // Your code here
+  enrollStudent(className, student) { 
+    let classObj = this.classes[className].students
+    let newStudent = new Student(student.name, student.age, student.city, student.grade)
```

Comment:
You've already created that `newStudent` in the app.js . You don't have to do it again here (or don't do it there).